### PR TITLE
Feature: Contact form

### PR DESCRIPTION
**Pull request by Rebelzob**
# Portfolio

- Added HTML and CSS form for the contact me section.
- Worked on pair programming.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -257,8 +257,47 @@ <h4 class="ab-titles skills"> Skills</h4>
         </div>
       </div>
     </section>
+    <section class="form-container flex-column">
+      <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      <div class="form flex-column">
+        <form class="flex-column " action="https://formspree.io/f/mayzgdra" method="POST" class="form-data">
+          <input type="text" id="name" placeholder="Full name" required maxlength="30">
+          <input type="email" id="email" placeholder="Email address" required>
+          <input type="text" id="message" placeholder="Write something..." required maxlength="500">
```
Comment (frempongdev) [inline index.html:266]: - [ ] Great implementation of your form and its field types. 👍🏿 
However, this `message` form input should have a field `type="textarea"`. Textarea types are more appropriate for more character inputs like messages and paragraphs. 📜

File: index.html [hunk 2]
```diff
@@ -257,8 +257,47 @@ <h4 class="ab-titles skills"> Skills</h4>
...
+          <center><button type="submit" class="button-resume">Get in touch</button></center>
+          <img src="/img/Rectangle 42.png" alt="" class="vector-footer">
+        </form>
```
Comment (frempongdev) [inline index.html:269]: You did a great job integrating **Formspree** in your form as an `action` in your form. 🚀 However, your form submission does not work as expected. **Formspree** requires a `name` attribute in order to match the input field.
 
image

- [ ] Please refer [here](https://formspree.io/html/) to check the proper implementation of **Formspree** forms. 🎯

File: styles.css [hunk 3]
```diff
@@ -415,4 +416,56 @@ a {
...
     border: 1px solid #3c3a39;
   }
+
+  .form-container {
+    background-color: #fff;
+  }
+
+  .form-container p {
+    font-family: 'Inter', sans-serif;
+    font-weight: 1000;
```
Comment (frempongdev) [inline styles.css:426]: - [ ] Please stick to the Figma design requirements as closely as possible as it is the main essence of this module. Paying attention to details is a good trait that can go a long way to satisfy your clients in your future job. 🏢 
Below is the recommended `font weight`. 

image

**Review by frempongdev — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

### Hi @Rebelzob, @MustafaSamiShirzad    👋🏼 
I'm Frempong.

You've done a great job implementing most of the requirements for this project 🚀, but there are a few changes I recommend be made. You're almost done, keep up the good work! 🤖 🎆 



 ## Highlights
✅ Great use of GitHub flow. 💯 
✅ No Linter errors. 🆗  
✅ Your project is documented well. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 
✅ Great UI. ✔️ 

 ## Required Changes ♻️
- [ ] Please check and remove the deployment section of your README as it is empty. 🗑️ 

_Kindly Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @Rebelzob and @MustafaSamiShirzad 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Implemented forms with HTML5 validations ✔️ 
Parsed a Figma design to create a UI ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @Rebelzob and @MustafaSamiShirzad 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Implemented forms with HTML5 validations ✔️ 
Parsed a Figma design to create a UI ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**